### PR TITLE
Pull data from Juju discourse

### PR DESCRIPTION
**Pull request by nottrobin**
Instead of the snapcraft one.
Also, switch over to sync workers for Talisker, and tidy up some requirements.

Fixes https://github.com/ubuntudesign/base-squad/issues/430

QA
--

`./run` - see data from discourse.jujucharms.com (even though it's still styled like snapcraft)

### CODE REVIEW

File: run [hunk 1]
```diff
@@ -155,12 +155,45 @@ while [[ -n "${1:-}" ]] && [[ "${1:0:1}" == "-" ]]; do
...
+        if ! docker inspect -f {{.State.Running}} ${db_container} &>/dev/null; then
+            docker run \
+                --name ${db_container}       `# Name the container`  \
+                --rm                         `# Remove the container once it's finished`  \
+                --volume "${db_volume}":/var/lib/postgresql/data  `# Store dependencies in a docker volume`  \
+                ${http_proxy}                `# Include HTTP proxy if needed`  \
+                --network ${network_name}    `# Use an isolated network`  \
+                --network-alias db           `# Call this container "db" on the network so it can be found`  \
+                --detach                     `# Run in the background` \
+                postgres                     `# Use the image for node version 7`
```
Comment (b-m-f) [inline run:183]: Comment is wrong here.

File: run [hunk 2]
```diff
@@ -155,12 +155,45 @@ while [[ -n "${1:-}" ]] && [[ "${1:0:1}" == "-" ]]; do
...
+            docker network create ${network_name}
+        fi
+
+        # Start the database
+        trap "kill_container ${db_container}" EXIT;
+        if ! docker inspect -f {{.State.Running}} ${db_container} &>/dev/null; then
+            docker run \
+                --name ${db_container}       `# Name the container`  \
+                --rm                         `# Remove the container once it's finished`  \
+                --volume "${db_volume}":/var/lib/postgresql/data  `# Store dependencies in a docker volume`  \
```
Comment (b-m-f) [inline run:178]: Saves PG data, not dependencies.

**Review by b-m-f — APPROVED** [review]

File: run (hunk 1 above)
Comment (nottrobin) [reply run:183]: Filed issue: https://github.com/canonical-webteam/generator-canonical-webteam/issues/105

File: run (hunk 2 above)
Comment (nottrobin) [reply run:178]: Mentioned in https://github.com/canonical-webteam/generator-canonical-webteam/issues/105

**Pull request merged**